### PR TITLE
Add NCSU

### LINKED ISSUE #4997: Add NCSU
- [X] I have searched both open and closed [issues](https://github.com/2factorauth/twofactorauth/issues). The issue I'm creating is not a duplicate of an existing issue.
- [X] I have searched both open and closed [pull requests](https://github.com/2factorauth/twofactorauth/pulls). The issue I'm creating is not a duplicate of an existing pull request.

### Information about the site to be added that supports 2FA: ###

* Site name - `NC State University`

* Site URL - `https://ncsu.edu`

* Does the site support 2FA? - `Yes`

* Documentation URL - `https://oit.ncsu.edu/it-security/2fa/`

### Information about what methods of 2FA the site supports: ###

* Software:
  - [X] TOTP (RFC-6238, Google Authenticator)
  - [X] Proprietary (Authy, Duo, etc.)

* Hardware:
  - [X] FIDO2/U2F security keys
  - [ ] Other

- [X] Phone calls
- [X] SMS tokens
- [ ] Email tokens

### Information about the site's eligibility to be added to twofactorauth.org: ###

- [X] I have checked that the site meets twofactorauth.org's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md).
- [X] The site is ranked within the top 200,000 sites on [Alexa](https://www.alexa.com/siteinfo/) or has a substantial social media following.
- [X] The site does not contain or promote content that violates twofactorauth.org's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md), including pornographic, discriminatory, or unlawful content.

### PR DESCRIPTION
**Pull request by billypoke**
Closes #4997

### CODE REVIEW

File: _data/education.yml [hunk 1]
```diff
@@ -397,6 +397,18 @@ websites:
...
+    url: https://ncsu.edu
+    img: ncsu.png
+    tfa:
+      - sms
+      - phone
+      - hardware
+      - proprietary
+      - totp
+      - u2f
+    doc: https://oit.ncsu.edu/it-security/2fa/
```
Comment (Carlgo11) [inline _data/education.yml:413]:
```suggestion
    regions:
      - us

```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Use this icon instead:
ncu

**Comment by billypoke** [comment]
Changes pushed

**Comment by Carlgo11** [comment]
I think you broke something in that last commit. Try reverting to the first one and redo it.

**Comment by billypoke** [comment]
Looks like line ending changes from my machine. Will correct that

**Comment by billypoke** [comment]
There we go

File: _data/education.yml [hunk 2]
```diff
@@ -397,6 +397,20 @@ websites:
...
     facebook: ScottishQualificationsAuthority
     img: mysqa.png
+    
+  - name: North Carolina State University
+    url: https://ncsu.edu
+    img: ncsu.png
+    tfa:
+      - sms
+      - phone
+      - hardware
```
Comment (phallobst) [inline _data/education.yml]: Maybe I didn't spot it, but which hardware 2FA method is used here (other than U2F keys)?
```suggestion
```

Comment (billypoke) [reply _data/education.yml]: Sounds good, thanks for all the helpful comments.

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
Thanks for the PR! :+1:

**Pull request merged**